client/components/account-status/account-fees.js

```diff
@@ -0,0 +1,80 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { __experimentalCreateInterpolateElement as createInterpolateElement } from 'wordpress-element';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrency, formatCurrency } from 'utils/currency';
+
+const AccountFees = ( props ) => {
+	const { accountFees } = props;
+	const baseFee = accountFees.base;
+	let currentFee = baseFee;
+
+	const currency = getCurrency( baseFee.currency );
+	let currencyCode = '';
+	if ( currency !== undefined ) {
+		currencyCode = currency.getCurrencyConfig().code;
+	}
```

Comment:
`baseFee.currency` is lower case version of the currency code already, so making it upperCase should be enough, no? I realized it from param name to `getCurrency` method.

Reply:
My thought here was that full currency name might also come from the config. It doesn't seem to be the case in #1375 so I might remove this line as per suggestion.